network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/FeederImpl.java

```diff
@@ -46,7 +46,7 @@ public Optional<String> getName() {
     }
 
     @Override
-    public ConnectablePosition.Feeder setName(String name) {
+    public ConnectablePosition.Feeder setName(String name, int side) {
```

Comment:
Same remark as above
In fact, this file should be removed, as Feeder interface is already implemented in ConnectablePositionImpl ...